index.html

```diff
@@ -208,6 +220,29 @@ <h2> Deliverables </h2>
         <div id="normative">
           <h3> Normative Specifications </h3>
           <dl class='todo'>
+            <dt class="spec"><a href="https://www.w3.org/TR/ttml3/">Timed Text
+                Markup Language 3 (TTML3)<br>
+              </a></dt>
+            <dd>This specification defines a content type that represents timed
+              text media for the purpose of interchange among authoring,
+              distribution and playback systems. Timed text is textual
+              information that is intrinsically or extrinsically associated with
+              timing information. </dd>
+            <dd>
+              <p class="draft-status"><b>Draft state:</b> <i><a href="https://w3c.github.io/ttml3/">Editor's
+                    Draft</a></i></p>
+              <p class="milestone"><b>Expected completion: [Q4-2019]</b> </p>
```

Comment:
Change "Q4-2019" to "Q3-2020". The goal is still the former, but we should build in some wiggle room.

Reply:
This is only setting an expectation, so there's adequate wriggle room already. I would rather not weaken that expectation right now. Happy to discuss on Thursday's call.